google-cloud-spanner/src/test/java/com/google/cloud/spanner/OptionsTest.java

```diff
@@ -163,10 +164,13 @@ public void listEquality() {
   @Test
   public void readOptTest() {
     int limit = 3;
-    Options opts = Options.fromReadOptions(Options.limit(limit));
+    String tag = "read";
+    Options opts = Options.fromReadOptions(Options.limit(limit), Options.tag(tag));
 
-    assertThat(opts.toString()).isEqualTo("limit: " + Integer.toString(limit) + " ");
-    assertThat(opts.hashCode()).isEqualTo(964);
+    assertThat(opts.toString())
+        .isEqualTo("limit: " + Integer.toString(limit) + " " + "tag: " + tag + " ");
+    assertThat(opts.tag()).isEqualTo(tag);
+    assertThat(opts.hashCode()).isEqualTo(3526226);
```

Comment:
not related to this change, but I noticed it now and added a small issue: #577

Reply:
Ok cool